### ISSUE TITLE
Pixel 2 (walleye): cannot install twrp permanently + boot issues

### ISSUE THREAD

**Issue by Irfy** [TeamWin/Team-Win-Recovery-Project#1104]
Using TWRP (3.2.0.0) for flashing various zips by temporarily booting it via fastboot seems to work as expected.

However, after installing the twrp installer zip (successfully, according to log output), restarting directly into recovery afterwards yields the default recovery with the error message "no command".

Restarting the phone (hold power button) hangs with the colorful G on white background (waited for several hours the first time, at least 2 minutes every subsequent time).

Restarting the phone again (hold power button) boots into the system properly.

Can I provide additional infos (what/how/where)?

**Comment by DigitalVortex92** [comment]
I also cannot get it to install permanently. On top of that, the App doesn't install once it is complete even though I tell it to. I've read somewhere that this is because the stock rom is replacing the recovery image. Could this be the case and is there any way to work around it. 

It seems the only persistent thing I can do is install magisk, but even that failed in my last attempt (no magisk app was installed). On top of that, I can't install things like Viper4Android when I did have magisk working. Persistent changes are not being kept on reboots :(

Edit: 
New Issue.
TWRP's Official app "Unable to locate recover" after install.
